### DISCUSSION TITLE
The poster - Govi-Nena Project

### DISCUSSION THREAD

**Discussion by chesadi** [RSEAA/RSEAA.github.io#64] [Posters for #RSEAA2023]
[V3_Poster_W.A.Indika.pdf](https://github.com/RSEAA/RSEAA.github.io/files/12604839/V3_Poster_W.A.Indika.pdf)

**Comment by rowlandm** [comment]
“Govi-Nena” mobile applications are gamechanging technological applications, specially designed and developed for Sri Lankan farmers and home gardeners - Nice!

**Reply by rowlandm** [reply]
I'd also like to know more about the software copyrights and the challenges you may have faced in that area :)

**Reply by chesadi** [reply]
We got the international US copyrights for the Apps. Long procedure. But, can do.
